public/components/event_analytics/explorer/visualizations/count_distribution/count_distribution.tsx

```diff
@@ -31,9 +40,62 @@ export const CountDistribution = ({ countDistribution }: any) => {
     },
   ];
 
+  // fill the final data with the exact right amount of empty buckets
+  function fillWithEmpty(processedData: any) {
```

Comment:
might be better to move this function outside and add tests. also could `processedData` have a type? `any` makes it difficult to reuse the function

Reply:
Is public/components/event_analytics/utils/utils.tsx a good file to have this function in or do you have somewhere else in mind?
I'll also make sure processData can have a type, but what is a good place to put that type definition?

Reply:
you can put it there. for the type maybe make the function more generic? for example it takes in `x` and `y` values instead of `processedData` which is a very specific single element tuple, and processed `intervalPeriod` instead of `selectedInterval`.

if the type of `processedData` is used across the project then you can probably leave it and add a type in some types.ts

Reply:
Made this change